edx_proctoring/tests/test_views.py

```diff
@@ -414,6 +414,7 @@ def setUp(self):
         self.student_taking_exam = User()
         self.student_taking_exam.save()
 
+        set_runtime_service('credit', MockCreditService())
```

Comment:
I guess this is to deal with credit_state.  It seems like this would have been required already for some existing tests, but maybe I'm missing something.  Anyway, I don't think this is an issue, I'm just trying to understand.